Source/RapyutaSimulationPlugins/Public/Core/RRGameSingleton.h

```diff
@@ -678,6 +678,8 @@ class RAPYUTASIMULATIONPLUGINS_API URRGameSingleton : public UObject
 
     static constexpr const TCHAR* TEXTURE_NAME_WHITE_MASK = TEXT("T_WhiteMask");
     static constexpr const TCHAR* TEXTURE_NAME_BLACK_MASK = TEXT("T_BlackMask");
+    static constexpr const TCHAR* TEXTURE_NAME_WHITE_COLOR_MASK = TEXT("T_WhiteColorMask");
+    static constexpr const TCHAR* TEXTURE_NAME_BLACK_COLOR_MASK = TEXT("T_BlackColorMask");
```

Comment:
Can we just replace old `T_WhiteMask, T_BlackMask` and keep their names here?

Reply:
OK